action.yml

```
@@ -1,14 +1,17 @@
-name: 'Release prep'
-description: 'Common setup for craft/prepare'
+name: "Release prep"
+description: "Common setup for craft/prepare"
 inputs:
   version:
     description: Version to release (optional)
     required: false
   force:
     description: Force a release even when there are release-blockers (optional)
     required: false
+  craft_version:
+    description: The craft version to use, defaults to latest
```

The `required` field is required by GitHub Actions as per [documentation](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_idrequired). It seems that the intention is that it is optional and defaults to `"latest"`.
```suggestion
    description: The craft version to use, defaults to "latest" (optional)
    required: false
    default: latest
```

---

My bad. I think we should still mark it as `required` as the default still kicks in in that case. We just don't want this to be null at any time.